src/screens/Home/HomePageBody.tsx

```diff
@@ -1,61 +1,70 @@
-import { Box, Typography } from "@mui/material";
+import { Box, Grid } from "@mui/material";
 import { Feed } from "features/feed/components";
 import useTheme from "features/theme/useTheme";
+import ConnectionList from "shared/components/ConnectionList";
+import { NavSidebar } from "shared/components/NavSidebar";
 import "./homePageBody.css";
 
+export const mockFriendList = [
+  {
+    id: 1,
+    email: "fdsjsdf@mfds.com",
+    firstName: "Khenen",
+    lastName: "Jacqso",
+    profilePic: "",
+  },
+  {
+    id: 2,
+    email: "fd113df@mfds.com",
+    firstName: "Rob",
+    lastName: "Bob",
+    profilePic: "",
+  },
+  {
+    id: 3,
+    email: "zaswaejsdf@mfd6s.com",
+    firstName: "Al",
+    lastName: "Ringer",
+    profilePic: "",
+  },
+];
+
 function HomePageBody() {
-  const {
-    isDarkMode,
-    themeColor: { color, backgroundColor },
-  } = useTheme();
+  const { isDarkMode, themeColor } = useTheme();
   return (
     <Box sx={{ bgcolor: isDarkMode ? "black" : "white" }}>
-      <Box
-        className="home-container"
-        sx={{
-          display: "flex",
-          justifyContent: "space-around",
-          p: "30px 30px 0 30px",
-          maxWidth: "1650px",
-          m: "auto",
-        }}
-      >
-        <Box className="left-sidebar" sx={{ maxWidth: "400px" }}>
-          <Box
-            sx={{
-              bgcolor: backgroundColor,
-              height: "350px",
-              borderRadius: "10px",
-              p: "20px",
-              color,
-            }}
-          >
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque
-            perspiciatis nulla facere ipsam praesentium illo incidunt fugiat.
-            Sit sunt rerum laudantium laboriosam perspiciatis consequuntur?
-            Molestiae quam quas provident inventore est!
-          </Box>
-        </Box>
-        <Box className="middle-feed" sx={{ ml: "15px", mr: "15px" }}>
+      <Grid container columns={24}>
+        <Grid
+          item
+          sm={5}
+          md={4}
+          lg={4}
+          xl={4}
+          sx={{ display: { xs: "none", sm: "flex" } }}
+        >
+          <NavSidebar theme={themeColor} />
+        </Grid>
+        <Grid
+          item
+          xs={24}
+          sm={19}
+          md={16}
+          lg={16}
+          xl={16}
+          sx={{ mt: "25px", pr: "20px", pl: "20px" }}
```

Comment:
I think you should copy the grid sizing me and Matt worked on in profile page. We got it decently responsive for a grid.